app/client/components/cancel/cancellationFlow.tsx

```diff
@@ -75,7 +79,7 @@ class ReasonPicker extends React.Component<
 
     // we extract the options from the children rather than direct from the ProductType to guarantee the routes are setup correctly
     const options = this.props.children.props.children.map(
-      (child: { props: MultiRouteableProps }) => child.props
+      (child: { props: GenericSaveAttemptProps }) => child.props
```

Comment:
This line seems slightly wild to me, but the children are `GenericSaveAttempt`s so this change is just so Typescript lets me access the `reason` prop.